app/src/main/java/com/besome/sketch/editor/PropertyActivity.java

```diff
@@ -0,0 +1,358 @@
+        Iterator<String> it3 = jC.b(this.q).d().iterator();
+        while (it3.hasNext()) {
+            for (Map.Entry<String, ArrayList<BlockBean>> entry : jC.a(this.q).b(it3.next()).entrySet()) {
+                Iterator<BlockBean> it4 = entry.getValue().iterator();
+                while (it4.hasNext()) {
+                    BlockBean next2 = it4.next();
+                    if ("setImage".equals(next2.opCode)) {
+                        if (m2.indexOf(next2.parameters.get(1)) < 0) {
+                            next2.parameters.set(1, "default_image");
+                        }
+                    } else if ("setBgResource".equals(next2.opCode) && m2.indexOf(next2.parameters.get(1)) < 0) {
+                        next2.parameters.set(1, "NONE");
+                    }
+                }
+            }
+        }
+    }
+
+    public void n() {
+        this.r.setVisibility(8);
+    }
+
+    public void o() {
+        ViewBean viewBean;
+        if (this.o.id.equals("_fab")) {
+            viewBean = jC.a(this.q).h(this.n.getXmlName());
+        } else {
+            viewBean = jC.a(this.q).c(this.n.getXmlName(), this.o.preId);
+        }
+        cC.c(this.q).a(this.n.getXmlName(), viewBean.clone(), this.o);
+        viewBean.copy(this.o);
+        Intent intent = new Intent();
+        intent.putExtra("view_id", this.o.id);
+        intent.putExtra("is_edit_image", this.p);
+        intent.putExtra("bean", viewBean);
+        setResult(-1, intent);
+        finish();
+    }
+
+    @Override // androidx.fragment.app.FragmentActivity
+    public void onActivityResult(int i, int i2, Intent intent) {
+        super.onActivityResult(i, i2, intent);
+        if (i == 209 && i2 == -1 && jC.d(this.q) != null && intent != null) {
+            String stringExtra = intent.getStringExtra("sc_id");
+            ArrayList<ProjectResourceBean> parcelableArrayListExtra = intent.getParcelableArrayListExtra("result");
+            if (jC.d(stringExtra) != null) {
+                jC.d(stringExtra).b(parcelableArrayListExtra);
+                m();
+            }
+        }
+    }
+
+    @Override // androidx.fragment.app.FragmentActivity
+    public void onBackPressed() {
+        this.u.i(this.o);
+        o();
+    }
+
+    @Override
+    // androidx.core.app.ComponentActivity, androidx.appcompat.app.AppCompatActivity, androidx.fragment.app.FragmentActivity, com.besome.sketch.lib.base.BaseAppCompatActivity
+    public void onCreate(Bundle bundle) {
+        super.onCreate(bundle);
+        setContentView(2131427627);
+        if (!super.j()) {
+            finish();
+        }
+        this.w = new ro(getApplicationContext());
+        this.k = (Toolbar) findViewById(2131231847);
+        a(this.k);
+        findViewById(2131231370).setVisibility(8);
+        d().d(true);
+        d().e(true);
+        this.k.setNavigationOnClickListener(v -> {
+            if (!mB.a()) {
+                Helper.getBackPressedClickListener(PropertyActivity.this);
```

Comment:
uhh what are you doing with that `View.OnClickListener`?

Reply:
idk, really it would work

Reply:
the helper method returns a `View.OnClickListener` but doesn't invoke `View$OnClickListener#onClick(View)` on it. that means the `Activity#onBackPressed` isn't getting invoked too. just leave the `onBackPressed();` call, the helper method is for other uses.